### ISSUE TITLE
Cannot install dependency on phoenix 1.5.7

### ISSUE THREAD

**Issue by zoedsoupe** [fewlinesco/bamboo_smtp#167]
When I add bamboo_smtp as dependency on `mix.exs` following the README or as: `{:bamboo_smtp, github: "fewlinesco/bamboo_smtp"}` and then run `mix deps.get`, I get this error:

```sh
Failed to use "ranch" (version 1.6.2) because
  cowboy (versions 2.7.0 and 2.8.0) requires ~> 1.7.1
  gen_smtp (version 1.0.1) requires 1.6.2
```

How I could proceed?

**Comment by arnaudmorisset** [comment]
Hello!

Thanks for reaching us. We will do some testing on our side, and we will come back once we have more information. 🙂

**Comment by arnaudmorisset** [comment]
Hello again 👋 

To fix this issue, `gen_smtp` maintainers need to update the version of `ranch` they required.

You can take a look at this issue: https://github.com/gen-smtp/gen_smtp/issues/234

**Comment by zoedsoupe** [comment]
Ok! Thanks for the attention! I'll address them to try to solve this! Thank you again!

**Comment by niccolox** [comment]
has this been fixed at gen_smtp? 

why is this issue closed?

**Comment by Hermanverschooten** [comment]
gen_smtp has released version 1.1.0 that fixes this, can you bump the dependency?

**Comment by LatitudeAdjustment** [comment]
I did not see this issue until now (status is closed for some reason).  
Please feel free to dump or use PR (#171) to address this.

Also note that going forward code will only work with OTP 20+, tests for OTP 19 and below will fail.

https://ninenines.eu/articles/cowboy-2.7.0/
